examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,825 @@
+{
```

Comment:
<ul><li>inherit from Sympy not override. We still have access to the default behaviors and properties with inheritance like for all out Sympy derived objects (Functino, ...)</li><li>"are passed as an argument to  ..." No space between Conditional and Dimension</li><li>A <code>ConditionalDimension</code> is used in two typical cases. (make bullet point list afterward)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>